### ISSUE TITLE
Different criterions 

### ISSUE THREAD

**Issue by rahulvigneswaran** [cc-hpc-itwm/GradVis#2]
Is it okay to use different criterions in the training phase and visualization phase? 
Also, what other criterions can be used ?

**Comment by keuperj** [comment]
@avraam-c can you comment on this? Did we do any additional experiments regarding this issue?

**Comment by avraam-c** [comment]
Yes this is possible, the training phase only serves as a way to get the weights of the network at different points during training. This is independent of the criterion used.
As long as the visualization function does not return an error, any criterion can be used for visualization e.g. one could train using Cross-Entropy loss and visualize using MSE loss etc.

**Comment by rahulvigneswaran** [comment]
@avraam-c Thank you so much for the reply. Hessian's Eigen spectrum is basically a tool to visualize loss landscape, so when I use Cross-Entropy in the training phase and MSE in the visualization phase, I will be given a loss-landscape for MSE or Cross-Entropy?

**Comment by avraam-c** [comment]
So basically the stochastic Lanczos method is a way to visualize the local loss landscape in every dimension by plotting the full Hessian eigenvalue spectrum, while the "visualize" function in this toolbox provides a way of looking at the loss landscape in two dimensions.
For visualization only the loss function in the visualization phase matters, so for example if you train using Cross-Entropy and then visualize using MSE, the loss landscape of the MSE loss will be shown. This applies for both methods, stochastic Lanczos as well as the 2D visualization.
